config/webpack.dev.js

```diff
@@ -90,7 +96,7 @@ module.exports = {
       // in development "style" loader enables hot editing of CSS.
       {
         test: /\.css$/,
-        loader: 'style!css!postcss'
+        loader: ExtractTextPlugin.extract('style-loader', 'css-loader', 'postcss-loader')
```

Comment:
What about having two commands?
`npm run dev` and `npm run dev:hmr` the second one disables the server side rendering